core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -68,6 +68,9 @@ final case class Kleisli[F[_], A, B](run: A => F[B]) { self =>
   def tapWith[C](f: (A, B) => C)(implicit F: Functor[F]): Kleisli[F, A, C] =
     Kleisli(a => F.map(run(a))(b => f(a, b)))
 
+  def toReader: Kleisli[Id, A, F[B]] =
```

Comment:
a generalization of this is `Kleisli[Lambda[x => F[G[x]]], A, B] => Kleisli[F, A, G[B]]`. I wonder if it is worth adding a method like that to the companion object:
```scala
def rotate[A, B, F[_], G[_]](k: Klesli[Lambda[x => F[G[x]]], A, B]): Klesli[F, A, G[B]] =
  k.mapF[F, G[B]](identity)
```
I don't know if it worth having the method or not to save one `mapF` call. `toReader` looks useful enough to me.